src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/GroovyCallSiteSelectorTest.java

```diff
@@ -92,6 +94,10 @@ public static void m2(long x) {}
     @Issue("JENKINS-45117")
     @Test public void constructorVarargs() throws Exception {
         assertEquals(EnvVars.class.getConstructor(), GroovyCallSiteSelector.constructor(EnvVars.class, new Object[0]));
+        Map<String, String> myMap = new HashMap<String, String>();
+        myMap.put("ONE", "one");
+        myMap.put("TWO", "two"); 
+        assertEquals(EnvVars.class.getConstructor(Map.class), GroovyCallSiteSelector.constructor(EnvVars.class, new Object[] {myMap}));
```

Comment:
This passes for me without the change to `GroovyCallSiteSelector`, so I'm not sure whether that change is actually relevant?

Reply:
Are you referring to the change in test? I know - it succeeds exactly becase of what I explained in my comment.
Do you think I should replace it with a test that uses PrarametersAction.class (which is what the fix was intended for?) It makes sense. I just didn't want to add additional dependencies.

Reply:
See my change over in #156 - I think it's better. =)

Reply:
Yes. looks like the proper way to fix this. 
Will build and test with our setup to verify.